components/Chat/Chat.tsx

```diff
@@ -284,6 +299,16 @@ export const Chat: FC<Props> = memo(
             />
           </>
         )}
+        {showScrollDownButton && (
+          <div className="absolute bottom-0 right-0 mb-4 mr-4 pb-20">
+            <button
+              className="flex h-7 w-7 items-center justify-center rounded-full bg-white shadow-md hover:shadow-lg focus:outline-none focus:ring-2 focus:ring-blue-500 dark:bg-[#515152d7]"
+              onClick={handleScrollDown}
+            >
+              <IconArrowDown className="h-4 w-4" />
```

Comment:
@dasunNimantha instead of classnames on the icon, try the size attribute.

Reply:
sure. will update